cli.py

```diff
@@ -117,6 +117,19 @@ def fetch_all_prices():
     write_prices(inspect_db_session, prices)
 
 
+@cli.command()
+@click.argument("start", type=int)
+@click.argument("end", type=int)
```

Comment:
`after` / `before`

`start` and `end` are subjective based on you perspective of time

Reply:
also i think click can deal with type datetime which would be nicer

Reply:
coingecko uses UNIX epoch for time, not datetime

Reply:
but people use datetime - not unix timestamp

Reply:
people, smh